kubejs/server_scripts/mods/buildinggadgets2.js

```diff
@@ -1,104 +1,19 @@
-ServerEvents.recipes( event => {
-    event.remove({id: /buildinggadgets2/})
-
-    //Template Manager
-    event.shaped(
```

Comment:
>  I'm fine with it being reverted to default recipes (except for the ones touched by HM recipe changes)

Trying to build off of @ThePansmith 's comment, this is reverting everything except the specific recipe change for HM.  With that said, does this sound right to y'all? Or is this too easy for HM?